libs/klv/src/main/java/org/codice/ddf/klv/KLV.java

```diff
@@ -0,0 +1,558 @@
+/**
```

Comment:
Question - did we change this class at all?  Looks like there is no particular licensing on this other than "public domain". http://iharder.sourceforge.net/current/java/klv/.